lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -124,8 +126,27 @@ const NotePopupModal = ({
 
   const hideDelete = !stripes.hasPerm('ui-notes.item.delete');
 
+  const titleContent = popupNote?.title && (
+    <div>
+      <strong><FormattedMessage id="stripes-smart-components.title" />: </strong>
+      {popupNote?.title}
+    </div>
+  );
+
   const popupContentHTML = { __html: popupNote?.content };
 
+  const detailsContent = popupNote?.content && (
+    <div className={styles['note-popup-details-container']}>
+      <p className={styles['note-popup-details-label']}>
+        <strong><FormattedMessage id="stripes-smart-components.details" />: </strong>
+      </p>
+      <span
+        data-test-note-popup-modal-content
```

Comment:
Please add class `ql-editor`. It makes sure that details are displayed in the same way as in create or edit page